rules/k8s_audit_rules.yaml

```diff
@@ -418,3 +418,110 @@
   priority: DEBUG
   source: k8s_audit
   tags: [k8s]
+
+
+
+- list: full_admin_k8s_users
+  items: ["admin", "kubernetes-admin",  "kubernetes-admin@kubernetes", "default", "kubernetes-admin@cluster.local", "minikube-user"]
```

Comment:
why `default` is admin user?

Reply:
K3s admin user is called "default", when /etc/rancher/k3s/k3s.yaml configuration file is created.

The best direct link I can find to share about this is: https://github.com/rancher/k3s/issues/140#issuecomment-469338111

Reply:
I don't know whether falco supports k3s ? And also whether the audit events from k3s is compatible with k8s. If you can provide more information here, that would be great!

Reply:
Maybe they are not, and we overstepped here. k3s have an audit webhook. Let me make some test but it will take some time.

Reply:
it may be good to remove `default` in this PR, once you confirm falco support k3s and k3s audit is event is compatible with k8s, then we can add it back.

Reply:
Remove the `default` user, and clarified comments on the rule to make clear that we are detecting only usernames and not roles.